### ISSUE TITLE
Add attribute to transformers to check if they should be tested for quality

### ISSUE THREAD

**Issue by amontanez24** [sdv-dev/RDT#296]
### Problem Description

By definition, some transformers don't do anything to help expose or preserve relationships in the data (eg. the `LabelEncodingTransformer`. This means they are very likely to fail quality tests.

As of now, the thresholds for quality tests have been lowered to account for this, but what would be better is if we could just skip any transformers that are known not to be good for quality in the tests.

### Proposal

1. Add an attribute and method to each transformer to determine if it is designed to be good for quality or not.
2. Raise the thresholds in the `tests/quality/test_quality.py` file.

**Comment by fealho** [comment]
I agree this should be revisited at some point, I just had to lower the threshold even further to allow the `CategoricalTransformer` to pass.
